main/end-to-end-tests/cypress/integration/appstream-egress-disabled/workspaces.spec.js

```diff
@@ -53,4 +62,27 @@ describe('Launch a workspace', () => {
     const workspaceName = launchWorkspace(emr, 'EMR');
     checkDetailsTable(workspaceName);
   });
+
+  it('should have opened a new tab for the RStudio Server when you click Connect', () => {
+    // Need to login at the beginning so the stub is initialized directly before we use to monitor the new tab opening
+    cy.login('researcher');
```

Comment:
I feel like this test case should be a function, and be called from `should launch a new RStudio Server workspace correctly`. This test case is dependent on the previous test running first and running correctly. It would be safer to establish that dependency by making this a test function.

Reply:
I did it this way to multitask and let the other tests run while the rstudio server is being created but I see what you mean. I'll combine them.

Reply:
Taking this part out because of the POST code change